src/client/public/css/components/separator.css

```diff
@@ -2,3 +2,9 @@
 	border: none;
 	border-top: 1px solid rgba(156, 156, 156, 0.68);
 }
+
+@media (max-width: 480px) {
+	.separator {
+		display: none;
+	}
+}
```

Comment:
we do not want all separators to disappear on the screen width
add an additional styles to separator component and apply these styles within sign up page